README.md

```diff
@@ -22,13 +22,13 @@ This repo is part of the [Microsoft Bot Framework](https://github.com/microsoft/
 
 Starting from Web Chat 4.6.0, Web Chat requires React 16.8.6 or up.
 
-Although we recommend that you upgrade your host app at your earliest convenience, we understand that host app may need some time before its React dependencies are updated, especially in regards to huge applications.
+Although we recommend you to upgrade your host app at your earliest convenience, we understand that host app may need some time before its react dependencies are updated, especially in regards to huge applications.
```

Comment:
I don't think this is right, `React` is a noun, so it must start by a CAPITAL letter.

Reply:
I agree with @callebdev.  I think both of your corrections to this sentence are unnecessary.